app/code/Magento/Inventory/composer.json

```diff
@@ -6,7 +6,6 @@
         "magento/framework": "100.2.*",
         "magento/module-inventory-api": "100.0.*",
         "magento/module-backend": "100.2.*",
-        "magento/module-catalog": "100.2.*",
```

Comment:
If you remove  the **magento/module-catalog** you must add dependency  for the **magento/module-inventory-catalog**